hw/dv/sv/alert_esc_agent/data/alert_esc_agent_testplan.hjson

```diff
@@ -0,0 +1,35 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  covergroups: [
+    {
+      name: alert_handshake_complete_cg
+      desc: '''If it is an alert agent, cover if the alert handshake completes.'''
```

Comment:
which block will include this testplan? should we split these into multiple testplan, so that the IP with alert agent imports the alert testplan, the IP with esc imports esc part.

should other regular IPs like uart import part of this testplan?

Reply:
Yeah that is a good point, probably better to split into different testplans :) Thanks